keyvalue.md

```diff
@@ -643,6 +664,9 @@ If any other error occurs, it returns an error.</p>
 </ul>
 <h4><a name="exists"><code>exists: func</code></a></h4>
 <p>Check if the key exists in the bucket.</p>
+<p>If the key exists in the bucket, it returns ok with true. If the key does
+not exist in the bucket, it returns ok with false.</p>
```

Comment:
In some places, you use "ok with false" and others you use "Ok(false)". I think I prefer the latter, but I'll defer to you. Please choose 1 variant and use it consistently.

Reply:
Good point. Will use `Ok(<bool>)` through the proposal.